tests/callbacks/test_progress_bar.py

```diff
@@ -146,9 +146,10 @@ def test_progress_bar_fast_dev_run(tmpdir):
     assert 1 == progress_bar.val_batch_idx
     assert 0 == progress_bar.test_batch_idx
 
-    # the main progress bar should display 2 batches (1 train, 1 val)
-    assert 2 == progress_bar.main_progress_bar.total
-    assert 2 == progress_bar.main_progress_bar.n
+    # the main progress bar should display 1 batch, as max_steps = 1
+    # when fast_dev_run=True
+    assert 1 == progress_bar.main_progress_bar.total
+    assert 1 == progress_bar.main_progress_bar.n
```

Comment:
this is indicating that this is a breaking change in the API

Reply:
I believe this is wrong and the previous behaviour seemed accurate. the main_progress should get 2 batches: (1 train, 1 val)